### PR TITLE
refactor(clean config)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

Remove unused config references pairs with: https://github.com/Clinical-Genomics/servers/pull/1123

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/models/cg_config.py
Comment (ChrOertlin) [inline cg/models/cg_config.py:1]: Removing these entries had no effect on the tests in cg at all... I might not understand correctly how these work.

Comment (henrikstranneheim) [reply cg/models/cg_config.py:1]: They seem unused except for DemultiplexConfig, 👍

**Review by henrikstranneheim — APPROVED** [review]
Looks great 👍

File: cg/models/cg_config.py [hunk 1]
```diff
@@ -306,6 +304,7 @@ class CGConfig(BaseModel):
     class Config:
         arbitrary_types_allowed = True
         fields = {
+            "cg_stats_api_": "cg_stats_api",
```
Comment (ChrOertlin) [inline cg/models/cg_config.py:307]:
```suggestion
```

**Comment by ChrOertlin** [comment]
# Tested with 
cg -l DEBUG clean remove-old-demultiplexed-run-dirs -d
cg backup archive-spring-files --dry-run

**Comment by ChrOertlin** [comment]
stage and prod:
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/stage/test_demux/221019_A00689_0663_BH7FYHDRX2/894614
done.
Log deploy... done.
cg, version 49.4.5

**Pull request merged**